test/unit/client/test_client.py

```diff
@@ -1072,7 +1072,7 @@ def check_downscope_token_request(
         mock_object_id,
         make_mock_box_request,
 ):
-    def do_check(item_class, scopes, additional_data, expected_data):
```

Comment:
Why did you change the order of these parameters?

Reply:
I changed the order here to match the signature of Client.downscope_token. Minor thing, but in that method, scopes comes first then the item. I thought it seems a little more consistent to match that signature in do_check